repository/s.json

```diff
@@ -5911,6 +5911,18 @@
 				}
 			]
 		},
+		{
+			"name": "SystemRDL",
+			"description": "Syntax Highlighting and Language Support for SystemRDL 2.0.",
```

Comment:
You can leave this description out of your entry here, package control will take the repo description from GitHub.

Reply:
Fixed, thanks

Reply:
It is? Maybe you need to push?